{"uid":"4c6bbfbd2f86b6bc","name":"TC 10.02.01 - Verify 'MEN'S DEALS' title is correct","fullName":"tests.test_sale.TestSideMenuBarMen#test_tc_10_02_01","historyId":"7d9cc06b830a248f9ded0f57b7152948","time":{"start":1737943552113,"stop":1737943557228,"duration":5115},"description":"Check title of Men's Deals section","descriptionHtml":"<p>Check title of Men's Deals section</p>\n","status":"skipped","statusMessage":"XFAIL This test is expecting failure == Assertion Error because of error in spelling Men's\n\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563272e2a7ca <unknown>\n#1 0x5632729222f0 <unknown>\n#2 0x563272972035 <unknown>\n#3 0x563272972251 <unknown>\n#4 0x5632729b8054 <unknown>\n#5 0x5632729969dd <unknown>\n#6 0x5632729b53b3 <unknown>\n#7 0x563272996753 <unknown>\n#8 0x563272963baa <unknown>\n#9 0x563272964dfe <unknown>\n#10 0x563272df538b <unknown>\n#11 0x563272df9307 <unknown>\n#12 0x563272de1e7c <unknown>\n#13 0x563272df9ec7 <unknown>\n#14 0x563272dc624f <unknown>\n#15 0x563272e192f8 <unknown>\n#16 0x563272e194c0 <unknown>\n#17 0x563272e29646 <unknown>\n#18 0x7f2c0309ca94 <unknown>\n#19 0x7f2c03129c3c <unknown>","statusTrace":"self = <test_sale.TestSalePage.TestSideMenuBarMen object at 0x7fcc4e08d910>\nsale_page = <pages.sale_page.SalePage object at 0x7fcc4f081b10>\n\n    @allure.title(\"TC 10.02.01 - Verify 'MEN'S DEALS' title is correct\")\n    @pytest.mark.xfail(reason=\"This test is expecting failure == Assertion Error \"\n                              \"because of error in spelling Men's\")\n    def test_tc_10_02_01(self, sale_page):\n        \"\"\"Check title of Men's Deals section\"\"\"\n>       title = sale_page.check_text_in_men_deals_title()\n\ntests/test_sale.py:65: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/sale_page.py:19: in check_text_in_men_deals_title\n    men_deals_title = self.element_is_visible(self.side_bar_locators.MEN_DEALS_TITLE)\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e0995c7e99076282a5e25aa7d490a205\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7fcc4e508cc0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x563272e2a7ca <unknown>\nE       #1 0x5632729222f0 <unknown>\nE       #2 0x563272972035 <unknown>\nE       #3 0x563272972251 <unknown>\nE       #4 0x5632729b8054 <unknown>\nE       #5 0x5632729969dd <unknown>\nE       #6 0x5632729b53b3 <unknown>\nE       #7 0x563272996753 <unknown>\nE       #8 0x563272963baa <unknown>\nE       #9 0x563272964dfe <unknown>\nE       #10 0x563272df538b <unknown>\nE       #11 0x563272df9307 <unknown>\nE       #12 0x563272de1e7c <unknown>\nE       #13 0x563272df9ec7 <unknown>\nE       #14 0x563272dc624f <unknown>\nE       #15 0x563272e192f8 <unknown>\nE       #16 0x563272e194c0 <unknown>\nE       #17 0x563272e29646 <unknown>\nE       #18 0x7f2c0309ca94 <unknown>\nE       #19 0x7f2c03129c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1737941851900,"stop":1737941851966,"duration":66},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder","time":{"start":1737941851966,"stop":1737941851966,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1737943550558,"stop":1737943551740,"duration":1182},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"sale_page","time":{"start":1737943551740,"stop":1737943552112,"duration":372},"status":"passed","steps":[{"name":"Open a browser","time":{"start":1737943551740,"stop":1737943552112,"duration":372},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Check title of Men's Deals section","status":"skipped","statusMessage":"XFAIL This test is expecting failure == Assertion Error because of error in spelling Men's\n\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563272e2a7ca <unknown>\n#1 0x5632729222f0 <unknown>\n#2 0x563272972035 <unknown>\n#3 0x563272972251 <unknown>\n#4 0x5632729b8054 <unknown>\n#5 0x5632729969dd <unknown>\n#6 0x5632729b53b3 <unknown>\n#7 0x563272996753 <unknown>\n#8 0x563272963baa <unknown>\n#9 0x563272964dfe <unknown>\n#10 0x563272df538b <unknown>\n#11 0x563272df9307 <unknown>\n#12 0x563272de1e7c <unknown>\n#13 0x563272df9ec7 <unknown>\n#14 0x563272dc624f <unknown>\n#15 0x563272e192f8 <unknown>\n#16 0x563272e194c0 <unknown>\n#17 0x563272e29646 <unknown>\n#18 0x7f2c0309ca94 <unknown>\n#19 0x7f2c03129c3c <unknown>","statusTrace":"self = <test_sale.TestSalePage.TestSideMenuBarMen object at 0x7fcc4e08d910>\nsale_page = <pages.sale_page.SalePage object at 0x7fcc4f081b10>\n\n    @allure.title(\"TC 10.02.01 - Verify 'MEN'S DEALS' title is correct\")\n    @pytest.mark.xfail(reason=\"This test is expecting failure == Assertion Error \"\n                              \"because of error in spelling Men's\")\n    def test_tc_10_02_01(self, sale_page):\n        \"\"\"Check title of Men's Deals section\"\"\"\n>       title = sale_page.check_text_in_men_deals_title()\n\ntests/test_sale.py:65: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/sale_page.py:19: in check_text_in_men_deals_title\n    men_deals_title = self.element_is_visible(self.side_bar_locators.MEN_DEALS_TITLE)\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e0995c7e99076282a5e25aa7d490a205\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7fcc4e508cc0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x563272e2a7ca <unknown>\nE       #1 0x5632729222f0 <unknown>\nE       #2 0x563272972035 <unknown>\nE       #3 0x563272972251 <unknown>\nE       #4 0x5632729b8054 <unknown>\nE       #5 0x5632729969dd <unknown>\nE       #6 0x5632729b53b3 <unknown>\nE       #7 0x563272996753 <unknown>\nE       #8 0x563272963baa <unknown>\nE       #9 0x563272964dfe <unknown>\nE       #10 0x563272df538b <unknown>\nE       #11 0x563272df9307 <unknown>\nE       #12 0x563272de1e7c <unknown>\nE       #13 0x563272df9ec7 <unknown>\nE       #14 0x563272dc624f <unknown>\nE       #15 0x563272e192f8 <unknown>\nE       #16 0x563272e194c0 <unknown>\nE       #17 0x563272e29646 <unknown>\nE       #18 0x7f2c0309ca94 <unknown>\nE       #19 0x7f2c03129c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Check text in Men's deals title","time":{"start":1737943552113,"stop":1737943557228,"duration":5115},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563272e2a7ca <unknown>\n#1 0x5632729222f0 <unknown>\n#2 0x563272972035 <unknown>\n#3 0x563272972251 <unknown>\n#4 0x5632729b8054 <unknown>\n#5 0x5632729969dd <unknown>\n#6 0x5632729b53b3 <unknown>\n#7 0x563272996753 <unknown>\n#8 0x563272963baa <unknown>\n#9 0x563272964dfe <unknown>\n#10 0x563272df538b <unknown>\n#11 0x563272df9307 <unknown>\n#12 0x563272de1e7c <unknown>\n#13 0x563272df9ec7 <unknown>\n#14 0x563272dc624f <unknown>\n#15 0x563272e192f8 <unknown>\n#16 0x563272e194c0 <unknown>\n#17 0x563272e29646 <unknown>\n#18 0x7f2c0309ca94 <unknown>\n#19 0x7f2c03129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/sale_page.py\", line 19, in check_text_in_men_deals_title\n    men_deals_title = self.element_is_visible(self.side_bar_locators.MEN_DEALS_TITLE)\n                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Find a visible element","time":{"start":1737943552113,"stop":1737943557227,"duration":5114},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563272e2a7ca <unknown>\n#1 0x5632729222f0 <unknown>\n#2 0x563272972035 <unknown>\n#3 0x563272972251 <unknown>\n#4 0x5632729b8054 <unknown>\n#5 0x5632729969dd <unknown>\n#6 0x5632729b53b3 <unknown>\n#7 0x563272996753 <unknown>\n#8 0x563272963baa <unknown>\n#9 0x563272964dfe <unknown>\n#10 0x563272df538b <unknown>\n#11 0x563272df9307 <unknown>\n#12 0x563272de1e7c <unknown>\n#13 0x563272df9ec7 <unknown>\n#14 0x563272dc624f <unknown>\n#15 0x563272e192f8 <unknown>\n#16 0x563272e194c0 <unknown>\n#17 0x563272e29646 <unknown>\n#18 0x7f2c0309ca94 <unknown>\n#19 0x7f2c03129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Find a present element","time":{"start":1737943552113,"stop":1737943557227,"duration":5114},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563272e2a7ca <unknown>\n#1 0x5632729222f0 <unknown>\n#2 0x563272972035 <unknown>\n#3 0x563272972251 <unknown>\n#4 0x5632729b8054 <unknown>\n#5 0x5632729969dd <unknown>\n#6 0x5632729b53b3 <unknown>\n#7 0x563272996753 <unknown>\n#8 0x563272963baa <unknown>\n#9 0x563272964dfe <unknown>\n#10 0x563272df538b <unknown>\n#11 0x563272df9307 <unknown>\n#12 0x563272de1e7c <unknown>\n#13 0x563272df9ec7 <unknown>\n#14 0x563272dc624f <unknown>\n#15 0x563272e192f8 <unknown>\n#16 0x563272e194c0 <unknown>\n#17 0x563272e29646 <unknown>\n#18 0x7f2c0309ca94 <unknown>\n#19 0x7f2c03129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', '.categories-menu > strong:nth-child(3) > span')"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', '.categories-menu > strong:nth-child(3) > span')"},{"name":"timeout","value":"10"}],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"821238e7f44fb312","name":"Screenshot","source":"821238e7f44fb312.png","type":"image/png","size":25649},{"uid":"a096c66aed1200b5","name":"HTML source","source":"a096c66aed1200b5.html","type":"text/html","size":258763}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1737943557330,"stop":1737943557394,"duration":64},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder::0","time":{"start":1737944335893,"stop":1737944335893,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"epic","value":"Sale Page"},{"name":"feature","value":"Testing Side Menu Bar - Men's Deals Section"},{"name":"tag","value":"@pytest.mark.xfail(reason='This test is expecting failure == Assertion Error because of error in spelling Men's')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_sale"},{"name":"subSuite","value":"TestSalePage"},{"name":"host","value":"fv-az1945-895"},{"name":"thread","value":"2794-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_sale"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":2,"passed":0,"unknown":0,"total":2},"items":[{"uid":"7fbed500b7252df","reportUrl":"https://verafes.github.io/luma_project/567//#testresult/7fbed500b7252df","status":"skipped","statusDetails":"XFAIL This test is expecting failure == Assertion Error because of error in spelling Men's\n\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55b5e80687ca <unknown>\n#1 0x55b5e7b602f0 <unknown>\n#2 0x55b5e7bb0035 <unknown>\n#3 0x55b5e7bb0251 <unknown>\n#4 0x55b5e7bf6054 <unknown>\n#5 0x55b5e7bd49dd <unknown>\n#6 0x55b5e7bf33b3 <unknown>\n#7 0x55b5e7bd4753 <unknown>\n#8 0x55b5e7ba1baa <unknown>\n#9 0x55b5e7ba2dfe <unknown>\n#10 0x55b5e803338b <unknown>\n#11 0x55b5e8037307 <unknown>\n#12 0x55b5e801fe7c <unknown>\n#13 0x55b5e8037ec7 <unknown>\n#14 0x55b5e800424f <unknown>\n#15 0x55b5e80572f8 <unknown>\n#16 0x55b5e80574c0 <unknown>\n#17 0x55b5e8067646 <unknown>\n#18 0x7f185a49ca94 <unknown>\n#19 0x7f185a529c3c <unknown>","time":{"start":1737857189039,"stop":1737857194153,"duration":5114}}]},"tags":["@pytest.mark.xfail(reason='This test is expecting failure == Assertion Error because of error in spelling Men's')"]},"source":"4c6bbfbd2f86b6bc.json","parameterValues":[]}